_config/html.yml

```
@@ -3,20 +3,276 @@ Name: corehtml
 ---
 SilverStripe\Core\Injector\Injector:
   SilverStripe\Forms\HTMLEditor\HTMLEditorConfig:
-    class: SilverStripe\Forms\HTMLEditor\TinyMCEConfig
-  SilverStripe\Forms\HTMLEditor\TinyMCEScriptGenerator: '%$SilverStripe\Forms\HTMLEditor\TinyMCECombinedGenerator'
-  SilverStripe\Forms\HTMLEditor\TinyMCECombinedGenerator:
-    class: SilverStripe\Forms\HTMLEditor\TinyMCECombinedGenerator
-    properties:
-      AssetHandler: '%$SilverStripe\Assets\Storage\GeneratedAssetHandler'
+    class: 'SilverStripe\Forms\HTMLEditor\TextAreaConfig'
```

```suggestion
    class: SilverStripe\Forms\HTMLEditor\TextAreaConfig
```
Don't need quotes for FQCN

---

It's just a string - there's nothing special about a FQCN that makes it a special non-quoted string - all YAML strings can have or omit quotes. It's a better practice to have them than to omit them IMO.

---

It's more than it doesn't have spaces so doesn't need spaces. We have loads of FCQN in our yml most without quotes that's all, better to be consistent

---

I think it's better to follow best practices for new code, even if existing code doesn't. That's one of the core philosophies behind how we apply the [coding conventions](https://docs.silverstripe.org/en/contributing/coding_conventions/).
Even though this isn't an explicit coding convention rule we have, adding quotes around strings is good practice.

> We have loads of FCQN in our yml most without quotes that's all

I  am 100% certain that's just because they're strings and strings don't strictly need quotes - not because there's anything special about them being FQCN.